custom_components/midea_ac_lan/fan.py

```diff
@@ -105,15 +99,8 @@ async def async_set_percentage(self, percentage: int) -> None:
             await self.hass.async_add_executor_job(self.set_percentage, percentage)
 
     def update_state(self, status: Any) -> None:
-        try:
+        if self.hass:
```

Comment:
What's the purpose of the if ?

Reply:
By BLE001: Do not catch blind exception: `Exception`

As far as I know, the only exception to `self.schedule_update_ha_state` is when `self.hass` is `None`.
and here is just a debug log, I think it's okay to remove it.

Or just change `Exception` to `AttributeError` ?

Do you have any advice?